next-app/app/strapi/page.tsx

```diff
@@ -0,0 +1,19 @@
+"use client"
+import {getForm} from "@/app/actions/getForm";
+// @ts-ignore
+
+import {FormBuilder} from "@guidesmiths/react-form-builder"
+
+const StrapiPage = async () => {
+    const {form} = await getForm()
+    return (
+        <div className="flex items-center justify-center max-h-full h-screen">
+            <div className="flex">
+                {form && <FormBuilder form={form} onSubmit={() => console.log("sent")}/>}
```

Comment:
Could you console.log the submitted data? Just to see that everything is coming through, nothing fancy.